application/src/main/java/org/togetherjava/tjbot/commands/moderation/WhoIsCommand.java

```diff
@@ -0,0 +1,136 @@
+            .appendDescription("\n**Is bot:** " + (user.isBot()))
+            .appendDescription(
+                    "\n**Registration date:** " + DATE_TIME_FORMAT.format(user.getTimeCreated()))
+            .setTimestamp(Instant.now())
+            .setFooter("Requested by " + event.getUser().getAsTag(),
+                    event.getMember().getEffectiveAvatarUrl());
+
+
+        event.replyEmbeds(embedBuilder.build())
+            .addActionRow(Button.of(ButtonStyle.LINK, "discord://-/users/" + user.getId(),
+                    "Click to see profile"))
+            .queue();
+
+    }
+
+    private static void handleWhoIsMember(final @NotNull SlashCommandEvent event,
+            final @NotNull Member member, final @NotNull User user) {
+
+        @SuppressWarnings("StringConcatenation")
+        EmbedBuilder embedBuilder = new EmbedBuilder()
+            .setAuthor(member.getEffectiveName(), member.getEffectiveAvatarUrl(),
+                    member.getEffectiveAvatarUrl())
+            .setThumbnail(user.getEffectiveAvatarUrl())
+            .appendDescription("**Mention:** " + user.getAsMention())
+            .appendDescription("\n**Tag:** " + user.getAsTag())
+            .appendDescription("\n**ID:** " + user.getId())
+            .appendDescription("\n**Is bot:** " + (user.isBot()))
+            .appendDescription(boostingMemberToString(member))
+            .appendDescription(
+                    "\n**Join date:** " + DATE_TIME_FORMAT.format(member.getTimeJoined()))
+            .appendDescription(
+                    "\n**Registration date:** " + DATE_TIME_FORMAT.format(user.getTimeCreated()))
+            .appendDescription("\n**Roles:** " + formatRoles(member))
+            .setColor(member.getColorRaw())
+            .setTimestamp(Instant.now())
+            .setFooter("Requested by " + event.getUser().getAsTag(),
+                    event.getMember().getEffectiveAvatarUrl());
+
+
+        event.replyEmbeds(embedBuilder.build())
+            .addActionRow(Button.of(ButtonStyle.LINK, "discord://-/users/" + user.getId(),
+                    "Click to see profile"))
+            .queue();
+    }
+
+    private static @NotNull String boostingMemberToString(final @NotNull Member member) {
+        OffsetDateTime timeBoosted = member.getTimeBoosted();
+        if (null != timeBoosted) {
+            return "\n**Is booster:** true" + "\n**Boosting since:** "
+                    + DATE_TIME_FORMAT.format(timeBoosted);
+        } else {
+            return "\n**Is booster:** false";
+        }
+    }
+
+    /*
+     * .addField("Mention:", user.getAsMention(), false) .addField("Tag:", user.getAsTag(), false)
+     * .addField("ID", user.getId(), false) .addField("Join date",
+     * DATE_TIME_FORMAT.format(member.getTimeJoined()), true) .addField("Registration date",
+     * DATE_TIME_FORMAT.format(user.getTimeCreated()), true) .addField("Roles", formatRoles(member),
+     * false)
+     */
```

Comment:
(Dont forget to delete)

Reply:
Yeah removed it locally, didn't push yet